From 56423b8e91e12a0d18fa52551b4d3e3cdcb94c14 Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Sun, 17 Jun 2018 12:17:35 +0200
Subject: [PATCH 25/28] SystemUI: Resolve status bar VPN icon tints

 * Remove the layout VPN icon's tint to respect
    SignalClusterView.java/applyIconTint's tint

 * Matches the handling of the Airplane Mode icon

 * Results: Before and after: https://ibb.co/eZ5Pud

 * Test: Bright wallpaper with status bar contrasts:
    https://ibb.co/dxupSy

Change-Id: I44cec84815b7c989a68eab45eeb762d5014bd0f3
---
 packages/SystemUI/res/layout/signal_cluster_view.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/packages/SystemUI/res/layout/signal_cluster_view.xml b/packages/SystemUI/res/layout/signal_cluster_view.xml
index 25c87d3f981..2e22943620d 100644
--- a/packages/SystemUI/res/layout/signal_cluster_view.xml
+++ b/packages/SystemUI/res/layout/signal_cluster_view.xml
@@ -32,7 +32,6 @@
         android:layout_width="wrap_content"
         android:paddingEnd="6dp"
         android:src="@drawable/stat_sys_vpn_ic"
-        android:tint="@color/background_protect_secondary"
         android:contentDescription="@string/accessibility_vpn_on"
         />
     <FrameLayout
-- 
2.17.1

